docs/installation/install-kubectl-karmada.md

```diff
@@ -26,6 +26,8 @@ wget https://github.com/karmada-io/karmada/releases/download/v0.9.0/kubectl-karm
 tar -zxf kubectl-karmada-linux-amd64.tar.gz
 ```
 
+> Latest kubectl-karmada release is saved using the suffix '.tgz'.
+
 Next, move `kubectl-karmada` executable file to `PATH` path, reference from [Installing kubectl plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#installing-kubectl-plugins).
```

Comment:
We changed the tarball suffix at v1.0.0, we don't need to note the change here(I'll update the release asset for release 1.0.0). 
Suggested changes:
```diff
diff --git a/docs/installation/install-kubectl-karmada.md b/docs/installation/install-kubectl-karmada.md
index 4ed8642..6809885 100644
--- a/docs/installation/install-kubectl-karmada.md
+++ b/docs/installation/install-kubectl-karmada.md
@@ -18,12 +18,12 @@ For installation instructions see [installing kubectl](https://kubernetes.io/doc

 Karmada provides `kubectl-karmada` plug-in download service since v0.9.0. You can choose proper plug-in version which fits your operator system form [karmada release](https://github.com/karmada-io/karmada/releases).

-Take v0.9.0 that working with linux-amd64 os as an example:
+Take v1.2.1 that working with linux-amd64 os as an example:
 
 ```bash
-wget https://github.com/karmada-io/karmada/releases/download/v0.9.0/kubectl-karmada-linux-amd64.tar.gz
+wget https://github.com/karmada-io/karmada/releases/download/v1.2.1/kubectl-karmada-linux-amd64.tgz
 
-tar -zxf kubectl-karmada-linux-amd64.tar.gz
+tar -zxf kubectl-karmada-linux-amd64.tgz
 ```

 Next, move `kubectl-karmada` executable file to `PATH` path, reference from [Installing kubectl plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#installing-kubectl-plugins).

```

Reply:
ok, i'll fix it